---
slug: nf-core-clipseq
title: "nf-core/clipseq 2.0 - a robust Nextflow pipeline for comprehensive CLIP data analysis"
url: https://github.com/nf-core/clipseq
image: ../../../images/visuals/2_13_charlotte.png
poster: ../../../images/posters/2_13_charlotte.pdf
poster_id: 13
speakers:
  - Charlotte Capitanchik
tags:
  - Software
  - Poster Group 2
---

<div className="mb-8">
  <small className="typo-small">
    Charlotte	Capitanchik, Chris Cheshire, Sam Ireland, D. Marc Jones, Rupert Faraway, Ira A. Iosub, Oscar G. Wilkins, Charlotte West, Nicholas M. Luscombe, Anob Chakrabarti, and Jernej Ule
  </small>
</div>

<hr className="border-t border-gray-50 mb-4 opacity-20" />

Crosslinking and immunoprecipitation (CLIP) technologies have become a central component of the molecular biologistsâ€™ toolkit to study protein-RNA interactions and thus to uncover core principles of RNA biology. There has been a proliferation of CLIP-based experimental protocols, as well as computational tools, especially for peak-calling. Consequently, there is an urgent need for a well-documented bioinformatic pipeline that enshrines the principles of robustness, reproducibility, scalability, portability and flexibility while embracing the diversity of experimental and computational CLIP tools. To address this, we present version 2.0 of nf-core/clipseq - a robust Nextflow pipeline for quality control and analysis of CLIP sequencing data. In v2.0 we offer enhanced motif analysis, grouped data analysis, summaries and metagene analysis among further advanced features. Alongside being available at https://nf-co.re/clipseq, users can freely analyse their data through the pipeline via an intuitive graphical user interface from https:/app.flow.bio.
