---
slug: nov-30-nanopore
timeframe: 11:30 - 11:45 PM (15 min)
title: "Nextflow and nanopores; democratising the analysis of long-read DNA sequences"
datetime: 2023-11-30T11:30:00.000Z
date: Nov 30, 2023
time: 11:30 PM
isChild: false
hasPage: true
speakers:
  - Stephen Rudd
tags:
  - Community
  - Sponsor
youtube:
youtubeUrl:
is_sponsor: true
---
<div className="mb-4">
  <small className="typo-small">
    Stephen Rudd
  </small>
</div>

<hr className="border-t border-gray-50 mb-4 opacity-20" />

We love Nextflow at Oxford Nanopore Technologies. Bioinformatics is integral to our goal of enabling the analysis of anything, by anyone, anywhere. This groovy workflow management system forms the functional backbone of our EPI2ME product.

EPI2ME provides a collection of best-practise bioinformatics workflows. These are designed to help our users both through their first steps in Nanopore sequence analysis and to democratise bioinformatics for users less familiar with the command line. We support a varied collection of workflows that enable tasks that vary from laboratory plasmid assembly to analysing single cell transcriptomics data. Or from analysing virus genomes to genetic variant calling from paired human tumour/normal DNA sequences.

In this presentation I will introduce how our workflows have evolved. Monolithic workflows using conda environments once made sense. Now we prefer the judicious usage of submodules and continue to explore how at (sequencing) run-time bioinformatics can enable refined decision making. Other themes to be introduced (albeit briefly) include the aliasing of workflows and our support for multiple architectures and platforms (Windows, macOS, ARM64, Graviton).

All of our workflows are freely available through GitHub with prebuilt containers distributed through DockerHub. Please see https://labs.epi2me.io for the project pages.

<div>
  <Button to="https://labs.epi2me.io" variant="secondary" size="md" arrow>
    View project
  </Button>
</div>
